api/src/pcapi/tasks/external_pro_tasks.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from pcapi import settings
+from pcapi.tasks.decorator import task
+from pcapi.tasks.serialization.external_pro_tasks import UpdateProAttributesRequest
+
+
+logger = logging.getLogger(__name__)
+
+EXTERNAL_PRO_QUEUE_NAME = settings.GCP_EXTERNAL_PRO_QUEUE_NAME
+
+
+# De-duplicate and delay by 15 minutes, to avoid collecting pro attributes and making an update request to Sendinblue
+# several times in a short time when a user managing an offerer makes several changes.
+#
+# Deduplication of Google tasks is based on identical payload in src.pcapi.tasks.cloud_task.enqueue_internal_task,
+# but Google documentation tells that "another task with the same name can't be created for ~1hour after the original
+# task was [...] executed."
+# Reference: https://cloud.google.com/tasks/docs/reference/rest/v2/projects.locations.queues.tasks/create
+#
+# So time_id parameter in UpdateProAttributesRequest helps to generate different hashes, so different task ids, every
+# 15 minutes. Keep delayed_seconds below consistent with time_id generation.
+@task(EXTERNAL_PRO_QUEUE_NAME, "/external/update_pro_attributes", True, 900)  # type: ignore [arg-type]
```

Comment:
En y reréflechissant je me demande si c'est pas plus smart de plutôt ajouter ça comme ça : 
```
def update_external_pro(email: str | None) -> None:
    # Call this function instead of update_external_user in actions which are only available for pro
    # ex. updating a venue, in which bookingEmail is not a User parameter
    from pcapi.tasks.sendinblue_tasks import update_pro_attributes_task
    from pcapi.tasks.serialization.sendinblue_tasks import UpdateProAttributesRequest

    if email:
        now = datetime.utcnow()
        update_sib_pro_attributes_task.delay(
            UpdateProAttributesRequest(email=email, time_id=f"{now.hour}:{now.minute // 15}")
        )
        update_beamer_pro_attributes_task.delay(
            UpdateProAttributesRequest(email=email, time_id=f"{now.hour}:{now.minute // 15}")
        )
```

Et du coup d'avoir 2 queues différentes. 
En terme d'avantage je me dit que si on veut limiter le nombre d'appels à beamer on pourrait augmenter le time_id juste pour beamer. 
Si une api répond beaucoup plus lentement que l'autre on ne ralentit pas l'autre queue.